.github/workflows/sonarqube-scan.yml

```diff
@@ -0,0 +1,23 @@
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+name: SonarQube scan
+jobs:
+  sonarqube_scan:
+    name: SonarQube Scan
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Scanning files using SonarQube
+        uses: sonarsource/sonarqube-scan-action@master
+        with:
+          args: >
+            -Dsonar.qualitygate.wait=true
+            -Dsonar.projectKey=composable-storefront
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          SONAR_HOST_URL: https://sonar.tools.sap
+    if: ${{ github.base_ref == 'develop' || startsWith(github.base_ref, 'release') }}
```

Comment:
quick question

even if it's not develop / release, wouldn't it be better that it's applied to any PRs whether or not it's the above base branch or not? 

it will help secure code quality instead of fixing things later once it gets into develop / release

Reply:
changed, will run on all PRs